clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -808,6 +808,31 @@ else if (max == null)
         }
     }
 
+    public static class ValidList implements Validator {
```

Comment:
I like this validator! Can you add a test for it in KafkaConfigTest near the others?